packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -2,156 +2,107 @@ import React, { FC, useState } from 'react';
 import { ItemRowFragment } from '@openmsupply-client/system';
 import {
   BasicSpinner,
-  useDialog,
   Divider,
-  TableContainer,
-  TabContext,
-  TabList,
-  Tab,
   useTranslation,
   useIsMediumScreen,
-  ButtonWithIcon,
-  PlusCircleIcon,
   Box,
   ModalMode,
   useNotification,
-  DialogButton,
 } from '@openmsupply-client/common';
-import { BatchTable, PricingTable } from './StocktakeLineEditTables';
-import { StocktakeLinePanel } from './StocktakeLinePanel';
 import { StocktakeLineEditForm } from './StocktakeLineEditForm';
 import { useStocktakeLineEdit } from './hooks';
-
+import {
```

Comment:
The stocktake modal in general:

- I removed the `StocktakeLinePanel` component as that was mainly adding in the batch/count this line columns and wasn't needed as I added those columns to the tables themselves.
- I added a `StocktakeLineModal` component to shift out that markup which was the bulk of it.
- I added a `StocktakeLineEditTabs` component to shift out the tabs markup which was a mix of what was in this file (the context and tabs) as well as what was in the `StocktakeLinePanel` (the 'tab container' styling)